river/InputConfig.zig

```diff
@@ -214,6 +214,35 @@ pub const ScrollButton = struct {
     }
 };
 
+pub const MapToOutput = struct {
+    output_name: ?[]const u8,
+
+    fn apply(map_to_output: MapToOutput, device: *wlr.InputDevice) void {
+        const output = out: {
+            if (map_to_output.output_name) |name| {
+                var it = server.root.active_outputs.iterator(.forward);
+                while (it.next()) |outp| {
+                    if (mem.eql(u8, mem.span(outp.wlr_output.name), name)) {
+                        break :out outp.wlr_output;
+                    }
+                }
+            }
+
+            break :out null;
+        };
+
+        var cur_node = server.input_manager.seats.first;
+        while (cur_node) |seat_node| {
+            log.debug("mapping input '{s}' -> '{s}'", .{
+                device.name,
+                if (output) |o| o.name else "<no output>",
+            });
+            seat_node.data.cursor.wlr_cursor.mapInputToOutput(device, output);
+            cur_node = seat_node.next;
```

Comment:
I'd rather hardcode input_manager.defaultSeat() for now and leave a `// TODO support multiple seats` comments than leave correct looking code that will cause wlroots to log a load of pointless errors.

Also, only input devices of type pointer, touch, and tablet_tool are supported for mapping to outputs and we don't support tablet_tool input devices at all yet. Please switch on `device.type` and ignore this setting if unsupported by the device.